.gitlab-ci.yml

```diff
@@ -435,11 +435,37 @@ check-polkadot-companion-build:
     - job:                         test-linux-stable-int
       artifacts:                   false
   script:
-    - ./.maintain/gitlab/check_polkadot_companion_build.sh
+    - ./.maintain/gitlab/check_companion_build.sh paritytech polkadot
   after_script:
     - cd polkadot && git rev-parse --abbrev-ref HEAD
   allow_failure:                   true
 
+check-cumulus-companion-build:
+  stage:                           build
+  <<:                              *docker-env
+  <<:                              *test-refs-no-trigger
+  needs:
+    - job:                         test-linux-stable-int
+      artifacts:                   false
+  script:
+    - ./.maintain/gitlab/check_companion_build.sh paritytech cumulus 'cargo check'
+  after_script:
+    - cd cumulus && git rev-parse --abbrev-ref HEAD
+  allow_failure:                   true
```

Comment:
was it for debug purposes? Processbot treats allow_failure as overridable by `bot merge force`

Reply:
It was simply this way in the polkadot-companion-build job and I didn't change it. I would actually also suggest we keep the cumulus + beefy companion jobs optional for a little while to ensure there are no problems, and then *later* make it a requirement.